client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -1271,6 +1278,16 @@ export const useCommandsArrayWpcom = ( {
 			},
 			icon: settingsIcon,
 		},
+		{
+			name: 'sendFeedback',
+			label: __( 'Send feedback' ),
```

Comment:
Can we add `Suggest command` as another matching keyword?

Also, it would be cool if we could automatically pipe these tickets into Slack...

Reply:
Added the `searchLabel` in: https://github.com/Automattic/wp-calypso/pull/85436/commits/e622aaca8d9372a677e1ea6a3b09a7c2703180d1